lang/python/borgbackup/Makefile

```
@@ -0,0 +1,63 @@
+#
+# Copyright (C) 2022 Julien Malik <julien.malik@paraiso.me>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=borgbackup
+PKG_VERSION:=1.2.2
+PKG_RELEASE:=1
+
+PYPI_NAME:=borgbackup
+PKG_HASH:=d730687443f1beb602b1d72bae36318f6f9654818fcdc50458540ec579e57260
+
+PKG_LICENSE:=BSD
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=The Borg Collective
+
+include $(TOPDIR)/feeds/packages/lang/python/pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include $(TOPDIR)/feeds/packages/lang/python/python3-package.mk
```

This should be for all packages like this:

```suggestion
include ../pypi.mk
include $(INCLUDE_DIR)/package.mk
include ../python3-package.mk
```

Not everyone has the same path as you.

---

ok. I overlooked this when submiting the PR, I needed this because it was from a separate feed I set up.

---

Nothing to worry about. ;-)